login.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <main class="login-main">
+    <div class="login-container">
+      <div class="login-image-container row">
+        <img src="assets/logo-wolox.png" alt="Logo Wolox" class="login-image-logo">
+      </div>
+      <form action="list.html" class="login-form">
+        <div class="column login-email-container">
+          <label for="login-email" class="login-label text-5">Email</label>
+          <input type="email" id="login-email" class="login-input border-radius-5">
+        </div>
+        <div class="column login-password-container">
+          <label for="login-password" class="login-label text-5">Password</label>
+          <input type="password" id="login-password" class="login-input border-radius-5">
+        </div>
+        <div class="btn-login-container">
+          <button type="submit" class="btn-login border-radius-5">Login</button>
+        </div>
```

Comment:
Same, you can set `margin-bottom` inside of your `btn-login` class. There is no need to set a container for this. The less divs we have the better